src/main/java/card/issuer/CardIssuers.java

```diff
@@ -0,0 +1,39 @@
+package main.java.card.issuer;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
+
+public class CardIssuers {
+    private static Map<String, CardIssuerParameters> cardIssuersMap =  new HashMap<String, CardIssuerParameters>();
+
+    public static boolean doesIssuerExist(String issuerName) {
```

Comment:
Checking with the `contains()` method seems sufficient
```suggestion
    public static boolean isIssuerExist(String issuerName) {
        return cardIssuersMap.contains(issuerName);
     }
```